app/main/views/signup.py

```diff
@@ -128,6 +129,12 @@ def create_application(token):
         application = data_api_client.create_application({'status': 'saved'})
         id = application['application']['id']
 
+        notification_message = 'By: {} ({})'.format(
+            current_user.name,
+            current_user.email_address
+        )
+        notify_team('A new seller has started an application', notification_message)
```

Comment:
might be better to do this at the end of the method after the create user (odds of an exception here are low but still possible)